src/content/docs/setup/configuration/commerce-configuration.mdx

```diff
@@ -22,42 +22,68 @@ When implementing your own project, you must update the configuration values wit
 <Vocabulary>
 ### Storefront configuration
 
-The `configs` spreadsheet contains the connection settings for your Commerce blocks. Each row in the sheet contains a `key` and a `value` that corresponds to a specific setting or usage in your Commerce backend. The `key` is used to retrieve the `value` from the `configs` spreadsheet and is used to connect your Commerce blocks to your Commerce backend.
+A JSON object which maps properties and values used by the Commerce boilerplate.
+
+The storefront configuration is a JSON object which lives in either [config-service](https://www.aem.live/docs/config-service-setup) and contains the connection settings for your Commerce blocks. The boilerplate repo also contains a `demo-config.json` file which can be renamed to `config.json` in lieu of using the config service.
```

Comment:
```suggestion
The storefront configuration is a JSON object which lives in either [config-service](https://www.aem.live/docs/config-service-setup) and contains the connection settings for your Commerce blocks. The boilerplate repo also contains a `demo-config.json` file that can be renamed to `config.json` instead of using the config service.
```